jupyter_run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. docker/env.sh
```

Comment:
use absolute path with $WORK_PATH that is defined while building docker image.
. $WORK_PATH/docker/env.sh